src/components/link/common/inline-link-common.ts

```diff
@@ -0,0 +1,25 @@
+import { type CSSResultGroup } from 'lit';
```

Comment:
It was probably due to the sequence of refactorings. It may have been a mixed kind of import at some point in the development.
Anyway, they are fixed now :)